tests/components/mazda/fixtures/get_ev_vehicle_status.json

```diff
@@ -0,0 +1,19 @@
+{
+    "chargeInfo": {
+        "lastUpdatedTimestamp": "20210807083956",
+        "batteryLevelPercentage": 80,
+        "drivingRangeKm": 218,
+        "pluggedIn": true,
+        "charging": true,
+        "basicChargeTimeMinutes": 30,
+        "quickChargeTimeMinutes": 15,
+        "batteryHeaterAuto": true,
+        "batteryHeaterOn": true
+    },
+    "hvacInfo": {
+        "hvacOn": true,
+        "frontDefroster": false,
+        "rearDefroster": false,
+        "interiorTemperatureCelsius": 15.1
+    }
+}
```

Comment:
Often it's preferably to include a newline at EOF (it makes diff / git diff and various other tools much happier)

Reply:
Added newlines at the end of the fixture files.